src/util/expander.js

```diff
@@ -31,16 +31,17 @@ export function expand(el, {
 	// set the property to the 'from' value
 	el.style[property] = from;
 
+	el.addEventListener('transitionend', function listener() {
```

Comment:
This prevents an occasional timing issue where the transition isn't quite finished, but the previous setTimeout is reached.

Reply:
Nice!